source/oic_labels.csv

```diff
@@ -0,0 +1,70 @@
+OIC Data Appendix,,,
```

Comment:
the oic data had a separate dataset consisting of an appendix of descriptions for each column in the dataset with the actual data... used this as variable labels